frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -135,64 +98,99 @@ const Group: React.FC<GroupProps> = ({
 }) => {
   const childrenList = React.Children.toArray(children);
 
+  const [openList, setListOpen] = React.useState(false);
+
+  const anchorRef = React.useRef(null);
+
+  const handleToggle = () => {
+    setListOpen(!openList);
+  };
+
+  const handleClose = event => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setListOpen(false);
+  };
+
+  let headingPath = window.location.pathname.replace("/", "").split("/")[0];
+
+  if (headingPath === "ec2") {
+    headingPath = "aws";
+  }
+
+  const isSelected = headingPath == heading.toLowerCase();
+
   // n.b. if a Workflow Grouping has no workflows in it don't display it even if
   // it's not explicitly marked as hidden.
   if (childrenList.length === 0) {
     return null;
   }
+
   return (
-    <List data-qa="workflowGroup">
-      <ListItem button onClick={() => updateOpenGroup(heading)}>
-        <GroupHeading color="primary" variant="h6">
-          {heading}
-        </GroupHeading>
-        <Grid container justify="flex-end" data-qa="toggle">
-          {open ? <ExpandLessIcon /> : <ExpandMoreIcon />}
-        </Grid>
-      </ListItem>
-      {!open ? <Divider variant="middle" /> : null}
-      <Collapse in={open} timeout="auto" unmountOnExit>
-        <List component="div" disablePadding>
-          {childrenList.map((c: React.ReactElement) => {
-            return React.cloneElement(c, { onClick: onNavigate });
-          })}
-        </List>
-      </Collapse>
-    </List>
+    <GroupList
+      data-qa="workflowGroup"
+      ref={anchorRef}
+      aria-controls={openList ? "workflow-options" : undefined}
+      aria-haspopup="true"
+      onClick={handleToggle}
+    >
+      <GroupListItem button onClick={() => updateOpenGroup(heading)} selected={isSelected}>
+        <GroupHeading align="center">{heading}</GroupHeading>
+        <Collapse in={open} timeout="auto" unmountOnExit>
+          <Popper
+            open={openList}
+            anchorEl={anchorRef.current}
+            role={undefined}
+            transition
+            placement="right-start"
+            style={{ zIndex: 1200 }}
```

Comment:
will elevate this to a prop, but I want to overlay the popper onto the drawer and the zindex of a drawer is 1200. https://material-ui.com/customization/z-index/